configure.ac

```diff
@@ -293,6 +293,21 @@ else
     AC_CONFIG_FILES(mtcr_ul/Makefile)
 fi
 
+# MLXREG
+AC_CHECK_LIB([expat], XML_ParserCreate_MM,
+		[AC_CHECK_HEADERS(expat.h, have_expat=true, have_expat=false)],
+		have_expat=false)
+
+if ! $have_expat; then
+AC_MSG_ERROR([Expat library is missing, please install expat xml parser to continue: expat-devel.]
+		[http://expat.sourceforge.net/])
+fi
+
+AC_LANG_PUSH([C++])
+AC_CHECK_HEADERS([boost/foreach.hpp], [],
```

Comment:
Boost dependency was only when enable_fw_mgr was set.
Now you added this dependency to all of the package.
What about users that used mstflint before but didn't have boost? It will break their build.

Maybe it's better to add new tools that require new dependencies as optional, with their own flag and check for dependencies only if the user chose to use this functionality?
I don't see a reason to force someone to install boost if he doesn't need the new functionality.

Also, what is the usage of boost in the new tools? Just foreach and regex?